releasenotes/notes/feature-sci-git-metadata-0b418d0a3529d11f.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    sci: Extracts and sends git metadata from environment variables ``DD_GIT_REPOSITORY_URL``, ``DD_GIT_COMMIT_SHA``,
```

Comment:
do we have any docs on docs.datadoghq.com we can point people to to learn more about the feature / configuration / etc?